src/software/ai/hl/stp/play/offense/corner_kick_play_test.py

```diff
@@ -0,0 +1,157 @@
+import sys
+
+import pytest
+import time
+
+import software.python_bindings as tbots
+from proto.play_pb2 import Play, PlayName
+from software.simulated_tests.ball_enters_region import *
+from software.simulated_tests.simulated_test_fixture import simulated_test_runner
+from proto.message_translation.tbots_protobuf import create_world_state
+from proto.ssl_gc_common_pb2 import Team
+
+# TODO issue  #2599 - Remove Duration parameter from test
+# @pytest.mark.parametrize("run_enemy_ai,test_duration", [(False, 20), (True, 20)])
+def test_corner_kick_play_bottom_left(simulated_test_runner):
+
+    # starting point must be Point
+    ball_initial_pos = tbots.Point(2.5, -3)
+
+    # Setup Bots
+    blue_bots = [
+        tbots.Point(-3, 2.5),
+        tbots.Point(-3, 1.5),
+        tbots.Point(-3, 0.5),
+        tbots.Point(-3, -0.5),
+        tbots.Point(-3, -1.5),
+        tbots.Point(4.6, -3.1),
+    ]
+
+    yellow_bots = [
+        tbots.Point(1, 0),
+        tbots.Point(1, 2.5),
+        tbots.Point(1, -2.5),
+        tbots.Field.createSSLDivisionBField().enemyGoalCenter(),
+        tbots.Field.createSSLDivisionBField().enemyDefenseArea().negXNegYCorner(),
+        tbots.Field.createSSLDivisionBField().enemyDefenseArea().negXPosYCorner(),
+    ]
+
+    # Game Controller Setup
+    simulated_test_runner.gamecontroller.send_ci_input(
+        gc_command=Command.Type.STOP, team=Team.UNKNOWN
+    )
+    #    simulated_test_runner.gamecontroller.send_ci_input(
+    #        gc_command=Command.Type.NORMAL_START, team=Team.BLUE
+    #    )
+    simulated_test_runner.gamecontroller.send_ci_input(
+        gc_command=Command.Type.DIRECT, team=Team.BLUE
+    )
+    simulated_test_runner.gamecontroller.send_ci_input(
+        gc_command=Command.Type.DIRECT, team=Team.BLUE
+    )
```

Comment:
why are we sending two DIRECT calls here?

Reply:
idk, it seemed more consistent with non-deterministic testing, but it might not be necessary